README.rst

```diff
@@ -289,7 +295,7 @@ When creating a new account they enforce some password rules like: length must b
 between 8 and 15 characters and there must be one upper case, one lower case letter
 and one number.
 
-|MetLife|
+| |MetLife|
```

Comment:
Please revert this to the original `|MetLife|`.

Reply:
:heavy_check_mark: